advocacy_docs/partner_docs/KastenbyVeeam/04-ConfiguringVeeamKasten.mdx

```diff
@@ -16,32 +16,32 @@ Implementing EDB Postgres for Kubernetes with Kasten by Veeam requires the follo
 - Kasten K10 installed on your system
 
 !!! Note
-    For this integration, use the **example.yaml** files provided in each section for the appropriate Kasten configuration pieces, and change any environment variables per your specific needs.
+    For this integration, use the `example.yaml` files provided for the appropriate Kasten configuration pieces, and change any environment variables per your specific needs.
     
-    The **Add the Backup Decorator Annotations to the Cluster** section is the important section for the Kasten addon integration. 
+    [Add the backup decorator annotations to the cluster](#add-the-backup-decorator-annotations-to-the-cluster) is important for the Kasten add-on integration. 
```

Comment:
This edit seemed a little less clear than the original text, even though that text is somewhat awkwardly worded.

Reply:
Agreed. Turned it into a proper reference. Thank you!